Evolutionary_loop/Loop_Parts/archived/Part_B_GPU_job2_asym.sh

```diff
@@ -1,45 +1,23 @@
 #!/bin/bash
-
 #*******************************************************************************
-#  File: Part_B_GPU_job1.sh
-#
-#	 This part of the loop:
-#     1. Prepares output.xmacro with generic parameters such as :
-#             I.   Antenna type
-#             II.  Population number
-#             III. Grid size
-#     2. Prepares simulation_PEC.xmacro with information such as:
-#             I.   Each generation antenna parameters
-#     3. Runs XF and loads XF with both xmacros. 
+#  Original file name: Part_B_GPU_job2_asym.sh
+#	 		This is Part B2 of the loop, which prepares and runs output.xmacro with 
+#			the relevant parameter (antenna type, population number, grid size, etc)
+#			Output.xmacro writes the XFDtd simulation data to an output file (.uan)
 #
 #  Programmer: OSU GENETIS Team
 #
 #  Revision history:
-#     07/11/23  Started modifying from the VPol version
+#     07/13/23  Jason Yao, third pass over the loop for training purposes
```

Comment:
Since we are using Git, maintaining a manual revision history is probably redundant.

Reply:
got it. I'll avoid writing too much of these in the future except when I merge several script into one and when
I rename scripts.